tests/test_fluent_version_marker.py

```diff
@@ -5,6 +5,11 @@ def test_dev_fluent_any():
     pass
 
 
+@pytest.mark.fluent_version("latest")
+def test_dev_fluent_latest():
+    pass
```

Comment:
Are these tests just to check if the "latest" and other markers exist? Think I missed these before